Gruntfile.js

```diff
@@ -152,12 +172,14 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks("grunt-contrib-uglify");
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks("grunt-env");
+  grunt.loadNpmTasks("grunt-exec");
   grunt.loadNpmTasks("grunt-http");
 
 
-  grunt.registerTask("build", ["clean:build", "env:build", "browserify:build", "uglify:build"]);
-  grunt.registerTask("dev", ["clean:dev", "connect:dev", "browserify:dev", "watch"]);
-  grunt.registerTask("design", ["clean:dev", "connect:dev", "browserify:design", "watch"]);
+  grunt.registerTask("build", ["clean:build", "env:build", "stylus:build", "browserify:build", "uglify:build"]);
```

Comment:
I know it is a "nitpicking" detail, so do if you think it worth, maybe creating a task:
```grunt.registerTask("dev-prepare", ["clean:dev", "connect:dev", "stylus:build"]);```
and then using it in "dev" and "design" so adding new tasks is easier.